fullUnicode has not been switched to prefer printable

Fix recent error in travis build. Description was:

  1) CharacterArbitrary
       fullUnicode [single unicode character]
         Should produce strictly smaller values along the shrink path:
     Property failed after 22 tests
{ seed: 1534464746637, path: "21:0:0:0" }
Counterexample: [[null,undefined],-1214002315,[0]]
Shrunk 3 time(s)
Got error: AssertionError: Expect "\u0000" to be strictly inferior to "&"

https://travis-ci.org/dubzzz/fast-check/jobs/417056792

Either a badly implemented test or a real bug.
To be checked

--- a/src/check/arbitrary/CharacterArbitrary.ts
+++ b/src/check/arbitrary/CharacterArbitrary.ts
@@ -102,7 +102,7 @@ function fullUnicode(): Arbitrary<string> {
   // It only produces valid UTF-16 code points
   const gapSize = 0xdfff + 1 - 0xd800;
   function mapping(v: number) {
-    if (v < 0xd800) return v;
+    if (v < 0xd800) return preferPrintableMapper(v);
     return v + gapSize;
   }
   // Do not call CharacterArbitrary or use fromCodePoint in it

